.github/workflows/alns.yml

```diff
@@ -17,9 +17,9 @@ jobs:
       matrix:
```

Comment:
The changes in this file fix a few warnings about deprecations in the CI runner.